### PR TITLE
[W4.4h][T09-4] Vincent Neo

### PR DESCRIPTION
**Pull request by tenvinc**
added new use case for LO W4.4h

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -77,6 +77,40 @@ Use case ends.
...
+=== Use case: Rename tags
+
+*MSS*
+
+. User requests to rename a tag
+. AddressBook shows a list of tags
+. User enters the tag to be changed
+. AddressBook prompts user for the new tag name
+. User enters the new tag name
+. Addressbook confirms the tag name with user
```
Comment (okkhoy) [inline docs/DeveloperGuide.adoc]: If you have this step, the next one should be something like `User confirms the rename operation`.
Remember, use cases depict the interaction between the user and the system

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -77,6 +77,40 @@ Use case ends.
...
+* 2a. The list is empty.
++
+Use case ends.
+
+* 3a. The given tag name is invalid.
+** 3a1. AddressBook shows an error message.
++
+Use case resumes at step 2.
+
+* 5a. There is no input by the user
```
Comment (okkhoy) [inline docs/DeveloperGuide.adoc]: I would expect the system to wait for the user input, instead of throwing an error message. (Or am I missing something!)

**Comment by okkhoy** [comment]
@tenvinc You can close this PR after reviewing the comments.

File: docs/DeveloperGuide.adoc (hunk 2 above)
Comment (tenvinc) [reply docs/DeveloperGuide.adoc]: I see. I was unclear with this step. I meant that the user accidentally presses enter which will provide an empty string to the system.

**Pull request closed without merging**